tests/conftest.py

```diff
@@ -0,0 +1,206 @@
+import json
+import os
+import time
+
+import pytest
+import requests
+
+from s3fs import S3FileSystem
+import uuid
+import socket
+from contextlib import closing
+
+
+@pytest.fixture()
+def file_a(bucket_names):
+    return bucket_names["test"] + "/tmp/test/a"
+
+
+@pytest.fixture()
+def file_b(bucket_names):
+    return bucket_names["test"] + "/tmp/test/b"
+
+
+@pytest.fixture()
+def file_c(bucket_names):
+    return bucket_names["test"] + "/tmp/test/c"
+
+
+@pytest.fixture()
+def file_d(bucket_names):
+    return bucket_names["test"] + "/tmp/test/d"
+
+
+@pytest.fixture()
+def s3_files():
+    return {
+        "files": {
+            "test/accounts.1.json": (
+                b'{"amount": 100, "name": "Alice"}\n'
+                b'{"amount": 200, "name": "Bob"}\n'
+                b'{"amount": 300, "name": "Charlie"}\n'
+                b'{"amount": 400, "name": "Dennis"}\n'
+            ),
+            "test/accounts.2.json": (
+                b'{"amount": 500, "name": "Alice"}\n'
+                b'{"amount": 600, "name": "Bob"}\n'
+                b'{"amount": 700, "name": "Charlie"}\n'
+                b'{"amount": 800, "name": "Dennis"}\n'
+            ),
+        },
+        "csv": {
+            "2014-01-01.csv": (
+                b"name,amount,id\n" b"Alice,100,1\n" b"Bob,200,2\n" b"Charlie,300,3\n"
+            ),
+            "2014-01-02.csv": (b"name,amount,id\n"),
+            "2014-01-03.csv": (
+                b"name,amount,id\n" b"Dennis,400,4\n" b"Edith,500,5\n" b"Frank,600,6\n"
+            ),
+        },
+        "text": {
+            "nested/file1": b"hello\n",
+            "nested/file2": b"world",
+            "nested/nested2/file1": b"hello\n",
+            "nested/nested2/file2": b"world",
+        },
+        "glob": {"file.dat": b"", "filexdat": b""},
+    }
+
+
+@pytest.fixture()
+def bucket_names():
```

Comment:
This and the file_* are not really fixtures, because they don't do any setup/teardown. Is it not simpler to leave them as constants? Is this the only way to avoid importing the values in test modules?